cmd/print.go

```diff
@@ -168,7 +168,7 @@ func confirm(w io.Writer, r io.Reader, message string, def bool) bool {
 		return def
 	}
 	input = strings.TrimSpace(strings.ToLower(input))
-	return (input == "y" || input == "yes") == def
+	return (input == "y" || input == "yes")
```

Comment:
curious why the comparison to the default was removed. Is there a logic that I'm not getting?